baselines/aad.md

```diff
@@ -628,19 +609,18 @@ The following implementation instructions that reference the AAD PIM service wil
 
 4.  Review the score for the action named **Use least privileged administrative roles.**
 
-5.  Review the **current score** value and compare it to the **max score**.
-If the current score is not the maximum value and the status is not **Completed**, you must perform the improvement actions. If that is the case, follow the detailed action steps and check the score again after 48 hours to ensure compliance.
+5.  Review the **current score** value and compare it to the **max score**. If the current score is not the maximum value and the status is not **Completed**, you must perform the improvement actions. If that is the case, follow the detailed action steps and check the score again after 48 hours to ensure compliance.
 
-#### MS.AAD.7.3v1 instructions:
-Performing a manual review of highly privileged users to determine which ones are not cloud-only is labor intensive; we recommend running the ScubaGear tool instead. ScubaGear will provide a list of the highly privileged users that are not cloud-only.
+#### MS.AAD.7.3v1 Instructions
+Performing a manual review of highly privileged users per the instructions below to determine which ones are not cloud-only is labor intensive; we recommend running the ScubaGear tool instead. ScubaGear will provide a list of the highly privileged users that are not cloud-only.
 
 1. Perform the steps below for each highly privileged role. We reference the Global Administrator role as an example.
 
 2. In **Azure Active Directory** select **Roles and administrators.**
 
 3. Select the **Global administrator role.**
 
-4. Under **Manage**, select **Assignments.** If you have AAD PIM, repeat the steps below for both the **Eligible** and the **Active** assignments. If a group is listed, you will need to determine the members of the group and perform the steps for each group member.
+4. Under **Manage**, select **Assignments.** If you have Azure AD PIM, repeat the steps below for both the **Eligible** and the **Active** assignments. If a group is listed, you will need to determine the members of the group and perform the steps for each group member.
 
 5. For each highly privileged user, execute the Powershell code below but replace the `username@somedomain.com` with the principal name of the user who is specific to your environment. You can get the data value from the **Principal name** field displayed in the Azure Active Directory portal.
```

Comment:
MS.AAD.7.3v1 Instructions; step 5 - there is a reference to 'Azure Active Directory portal' which is inconsistent with word doc where its referred to 'Azure AD portal'

Reply:
Fixed.